views/blacklist.ejs

```diff
@@ -1,278 +1,79 @@
-                                data-bs-toggle="modal" data-bs-target="#deleteBlackList3">
-                                <i class="bi bi-trash"></i>
-                            </button>
-                        </div>
-                    </div>
-                    <div class="modal fade" id="deleteBlackList3" tabindex="-1" aria-labelledby="deleteBlackList3"
-                        aria-hidden="true">
-                        <div class="modal-dialog modal-dialog-centered">
-                            <div class="modal-content">
-                                <div class="modal-header">
-                                    <h2 class="modal-title fs-5" id="deleteBlackList3">Weet je zeker dat je deze
-                                        quote uit je blacklist wil verwijderen?</h2>
-                                </div>
-                                <div class="modal-body">
-                                    <p>It's the tree. Gandalf! Gandalf!</p>
-                                </div>
-                                <div class="modal-footer">
-                                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal"
-                                        aria-label="Close">Behouden</button>
-                                    <button type="button" class="btn btn-primary" formmethod="POST"
-                                        formaction="/">Verwijderen</button>
+                        <!-- Confirm delete window -->
+                            <div class="modal fade" id="deleteBlackList<%= index + 1 %>" tabindex="-1" aria-labelledby="deleteBlackList1" aria-hidden="true">
+                                <div class="modal-dialog modal-dialog-centered">
+                                    <div class="modal-content">
+                                        <div class="modal-header">
+                                            <h2 class="modal-title fs-5" id="deleteBlackList1">Weet je zeker dat je deze
+                                                quote uit je blacklist wil verwijderen?</h2>
+                                        </div>
+                                        <div class="modal-body">
+                                            <p><%= item.dialog %></p>
+                                        </div>
+                                        <div class="modal-footer">
+                                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal"
+                                                aria-label="Close">Behouden</button>
+                                            <form action="/blacklist/<%= item.quote_id %>/delete" method="post">
+                                                <button type="button" class="btn btn-primary" formmethod="post"
+                                                    formaction="/blacklist/<%= item.quote_id %>/delete">Verwijderen</button>
+                                            </form>
```

Comment:
"formaction" and "formmethod" should be removed on the button, since <form> is used instead. So:

```suggestion
                                            <form action="/blacklist/<%= item.quote_id %>/delete" method="post">
                                                <button type="button" class="btn btn-primary">Verwijderen</button>
                                            </form>
```